kolibri/core/assets/src/api-resource.js

```diff
@@ -289,6 +300,15 @@ class Collection {
   }
 
   /**
+   * Clear this Collection's cache of models.
+   */
+  clearCache() {
+    // Reset current models.
+    this.models = [];
+    this._model_map = {};
```

Comment:
My intuition is 'yes', since the local data is liable to be stale at that point? Also just because we do it for malformed data, and they seem like similar situations?